modules/mill-codegen-plugin/test/src-mill-0_12/smithy4s/codegen/mill/Smithy4sModuleSpec.scala

```diff
@@ -144,6 +144,36 @@ class Smithy4sModuleSpec extends munit.FunSuite {
     }
   }
 
+  test("2.13 codegen with placeholder wildcards") {
+    object foo extends TestBaseModule with Smithy4sModule {
+      override def scalaVersion = "2.13.16"
+      override def ivyDeps = Agg(coreDep)
+      override def scalacOptions =
+        Seq("-Xsource:3", "-P:kind-projector:underscore-placeholders")
```

Comment:
oh yes, obviously fails with ` bad option: -P:kind-projector:underscore-placeholders` due to no plugin in scope 🤔